frontend/packages/ceph-storage-plugin/src/types.ts

```diff
@@ -153,6 +153,39 @@ export type KMSConfig = {
   hasHandled: boolean;
 };
 
+export type IbmKmsConfig = {
+  name: {
+    value: string;
+    valid: boolean;
+  };
+  instanceId: {
+    value: string;
+    valid: boolean;
+  };
+  apiKey: {
+    value: string;
+    valid: boolean;
+  };
+  rootKey: {
+    value: string;
+    valid: boolean;
+  };
+  baseUrl: string;
+  tokenUrl: string;
+  hasHandled: boolean;
+};
+
+export enum ProviderNames {
+  vault = 'vault',
+  ibmKms = 'ibmKms',
```

Comment:
What about 
```
export enum ProviderNames {
  VAULT = 'VAULT',
  IBMROCKS = 'IBMROCKS',
 }
(or)
export enum ProviderNames {
  VAULT = 'Vault',
  IBMROCKS = 'IbmRocks',
 }
 ```
 
 Is it ibmkms (or) ibmrocks?

Reply:
ack... will change it...